ui/ops/src/dynamic/widgets/meter/MeterHistoryCard.vue

```diff
@@ -126,8 +144,11 @@ const nameForDescribe = computed(() => {
   if (props.subProductionNames.length > 0) return toName(props.subProductionNames[0]);
   return undefined;
 })
+
+const metricType = ref('unscaled');
 const {response: meterInfo} = useDescribeMeterReading(nameForDescribe);
 const unit = computed(() => meterInfo.value?.usageUnit);
+const displayUnit = computed(() => metricType.value === 'scaled' ? props.scaleUnit : unit.value);
```

Comment:
I'm not sure you got what I was asking. There's no reason to distinguish between displayUnit and unit. Remove this const and update the `unit` const to look more like this:

```js
const unit = computed(() => {
  const base = meterInfo.value?.usageUnit;
  if (base && metricType.value === 'scaled') {
    return `${base}/${props.scaleUnit}`;
  }
  return base;
})
```